plugins/caches.js

```diff
@@ -332,6 +333,96 @@ exports.plugin = {
                     }
                 }
             }
+        }, {
+            method: 'DELETE',
+            path: '/caches/{scope}/{id}',
+            handler: async (request, h) => {
+                if (strategyConfig.plugin !== 's3') {
+                    return h.response();
+                }
+
+                let cachePath;
+                const apiUrl = config.get('ecosystem.api');
+                const payload = {
+                    url: `${apiUrl}/v4/isAdmin`,
+                    method: 'GET',
+                    headers: {
+                        Authorization: `Bearer ${request.auth.token}`,
+                        'Content-Type': 'application/json'
+                    },
+                    json: true
+                };
+
+                switch (request.params.scope) {
+                case 'events': {
+                    return h.response();
+                }
+                case 'jobs': {
+                    const jobIdParam = request.params.id;
+
+                    payload.qs = {
```

Comment:
Is this the same as query param?
Also, I think this configuration is removed from hapi 12: https://github.com/hapijs/hapi/issues/2985

Reply:
fixed

Reply:
wait this is not hapi payload, this is the payload argument to the npm requests module https://www.npmjs.com/package/request#requestoptions-callback So qs is right